data/formats-data.js

```diff
@@ -4483,7 +4490,7 @@ exports.BattleFormatsData = {
 		tier: "PU",
 	},
 	lillipup: {
-		randomBattleMoves: ["return", "wildcharge", "firefang", "crunch", "icefang"],
+		randomBattleMoves: ["return", "playrough", "firefang", "crunch", "psychicfangs", "thunderwave"],
```

Comment:
thunderwave isnt needed

Reply:
but with that mediocre speed tier + the general usefulness of para i definitely see application

Reply:
lillipup has pretty much no bulk (lol) it's better to get the coverage over twave

Reply:
fien